packages/os/etc-systemd-resolved.conf.d.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=systemd-resolved drop-in directory
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=selinux-policy-files.service
+Wants=selinux-policy-files.service
+
+[Mount]
+What=tmpfs
+Where=/etc/systemd/resolved.conf.d
+Type=tmpfs
+Options=nosuid,nodev,noexec,noatime,context=system_u:object_r:etc_t:s0,mode=0755
```

Comment:
We found that using `ExecStartPre` to do the `mkdir` isn't tenable because `systemd-resolved` has `ProtectSystem=strict` in its unit file.  This setting mounts the entire filesystem read-only, preventing the `mkdir`.

We will opt to create these directories in `netdog` and avoid the units altogether.